lib/galaxy/config/sample/reports.yml.sample

```diff
@@ -52,7 +52,7 @@ uwsgi:
 
   # Mount the web application (e.g. Galaxy, Reports, etc.) at the given
   # URL prefix. Cannot be used together with 'module:' above.
-  #mount: /galaxy=galaxy.webapps.galaxy.buildapp:uwsgi_app()
+  #mount: /galaxy=galaxy.webapps.reports.buildapp:uwsgi_app()
```

Comment:
Ran into this one when setting up reports at a prefix; the example here (unlike the regular module line) is incorrect for reports. I just realized, though, that it will get overwritten by a make config-rebuild, but I don't immediately see where I can contextualize this reports default differently from the galaxy config -- anyone know off the top of their head?

Reply:
Took the liberty to just push that right to your branch in https://github.com/galaxyproject/galaxy/pull/10037/commits/9487f5f541a5255dbe2075b787de4db5936845ea, hope that's ok.

Reply:
@mvdbeek Always totally fine, thanks!  I forgot to come back to this one.